### ISSUE TITLE
Custom message to unregistered users

### ISSUE THREAD

**Issue by jangevaare** [Asgaros/asgaros-forum#102]
Another feature request here. Would it be possible to write custom messages when unregistered members attempt to access a member-only forum? I've manually done this a few times, and that's not a problem, but I wonder how difficult it would be to build into your software. In my case I have a front end login/registration page that I'd prefer to direct users to rather than the backend login page. I'd also like to display a custom message. Thanks!

**Comment by Asgaros** [comment]
Hello @jangevaare 

So as far as I understand you want to change the following text which appears when the option "Forum visible to logged in users only" is activated, because you want to have another URL for the login-link:
_"Sorry, only logged in users have access to the forum. » Login"_

Am I correct so far?

Have you tried to use the following WordPress core filter so it shows to your custom login-page instead of the default one?

https://codex.wordpress.org/Plugin_API/Filter_Reference/login_url

**Comment by jangevaare** [comment]
Correct, though I don't only want change the URL, I'd also like to change the message that is shown.

**Comment by Asgaros** [comment]
Hello again, @jangevaare 

I added a new filter for this, so you can easily change the complete error-message. It is called:
`asgarosforum_filter_error_message_require_login`

You can use it like this:
```
function my_login_message($message) {
	return 'My new error message. <a href="#">Login</a>';
}
add_filter('asgarosforum_filter_error_message_require_login', 'my_login_message');
```

This commit will be included in the upcoming v1.3 release.
